src/main/java/frc/robot/subsystems/Vision/VisionIO.java

```diff
@@ -15,12 +15,11 @@ public interface VisionIO {
   public static class VisionIOInputs {
     public double timestamp = 0.0;
     public double latency = 0.0;
-    public List<PhotonTrackedTarget> targets =
+    public List<PhotonTrackedTarget> coprocPNPTargets = // TODO is this a bad name
```

Comment:
i mean this isnt coproc pnp, this is just targets

Reply:
?? so what does [this](https://github.com/HighlanderRobotics/Crescendo/pull/25#discussion_r1458111913) mean then